src/index.html

```diff
@@ -140,12 +162,61 @@ <h2 class="modal__title">New Post</h2>
           <button
             aria-label="close modal"
             type="button"
-            class="modal__close modal__close_preview"
+            class="modal__close modal__close_offset"
           ></button>
           <img class="modal__image" src="#" alt="" />
           <p class="modal__caption"></p>
         </div>
       </div>
     </div>
+
+    <!-- Avatar modal -->
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close">
+          <img src="<%=require('./images/close-icon.svg')%>" />
+        </button>
+        <h2 class="modal__title">Edit profile avatar</h2>
+        <form name="edit-avatar-form" class="modal__form" novalidate>
+          <label for="profile-avatar-input" class="modal__label">
+            Profile Picture
+            <input
+              id="profile-avatar-input"
+              type="text"
```

Comment:
`  type="text"`


* Please, add `type="url"` for the `avatar` input, to validate it as a `link` https://snipboard.io/g9uSzU.jpg 


```html

            <input
              id="profile-avatar-input"
              type="url" 
              class="modal__input"
              placeholder="Paste the link to an image"
              required
            />
```